requirements.txt

```diff
@@ -1,11 +1,12 @@
 cookiecutter
 coverage
 detect-secrets==1.0.3
+#pytest-cookies
```

Comment:
Should this be uncommented or deleted?

Reply:
Commented as once the fix come in for pytest-cookie it will replace the forked document. I left it commented as an easy way to revert once update has happened. Happy to remove if you think otherwise

Reply:
No that's fine. Do we have an issue for reverting this once pytest-cookie is updated?

Reply:
Not yet but I will add one